client/components/InitiateCycle/index.jsx

```diff
@@ -269,8 +269,8 @@ class InitiateCycle extends Component {
 
         return (
             <Fragment>
-                <h2 data-test="initiate-cycle-h2">Initiate a Test Cycle</h2>
-                <h3 data-test="initiate-cycle-h3">Test Cycle Configuration</h3>
+                <h1 data-test="initiate-cycle-h2">Initiate a Test Cycle</h1>
```

Comment:
Now the test tags don't match the elements: 

```
h1 data-test="initiate-cycle-h2"
^^                           ^^
```

Reply:
Resolved.